test/test_index.js

```diff
@@ -27,13 +27,13 @@ describe('runModel', function() {
     global.odin = [ model ];
 
     const mm = getMixingMatrix('Nigeria');
-    const beta = getBeta('Nigeria');
+    const beta = estimateBeta('Nigeria', [3, 3/2, 3]);
     runModel(
       getPopulation('Nigeria'),
-      [0, 50, 100],
-      [mm, mm, mm],
+      [0],
```

Comment:
I don't understand why the arrays aside from `[mm]` have become single elements here. `beta` I guess is now returned as a vector from `estimateBeta` because we have vector input, but the `[0, 50, 100]` - should that not stay as a vector?

Reply:
in the first scenario we were changing the mixing matrix to the same matrix at t = [0, 50, 100].

That niggled at me a bit. So I changed it to simply set the mixing matrix at t = 0.

And yes, the tt_beta is still a vector, it's given on line 35. It's a bit harder to see without named arguments in js!